src/mocks/handlers/getProductListHandler.ts

```diff
@@ -0,0 +1,42 @@
+import { HttpResponse, http } from 'msw';
+
+import { API_URL } from '@/api/config';
+import { ENDPOINT } from '@/api/endpoints';
+import fashionProductListWith20Items from '@/mocks/data//productList/category/20ItemsFashion.json';
+import fashionProductListWith4Items from '@/mocks/data/productList/category/4ItemsFashion.json';
+import fitnessProductListWith20Items from '@/mocks/data/productList/category/20ItemsFitness.json';
```

Comment:
실제 파일명과 import문에서의 이름이 서로 달라보여 확인 부탁드려봅니당

```suggestion
import fitnessProductListWith20Items from '@/mocks/data/productList/category/20itemsFitness.json';
```